types/index.d.ts

```diff
@@ -107,7 +107,7 @@ export function combine(...hooks: Hook[]): Hook;
  * Display the current hook context for debugging.
  * {@link https://feathers-plus.github.io/v1/feathers-hooks-common/index.html#Debug}
  */
-export function debug(): Hook;
+export function debug(msg: string, fieldNames?: string[]): Hook;
```

Comment:
fieldNames is a rest param, please use `...fieldNames`. Could you please fix the test, too?

Thanks! :)

Reply:
Done!